### ISSUE TITLE
How to test stories for custom React ErrorBoundary component

### ISSUE THREAD

**Issue by diginikkari** [storybookjs/test-runner#218]
I'm trying to run smoke tests for stories including custom ErrorBoundary component.

When running storybook smoke tests. It is failing when there is story for ErrorBoundary with children throwing error.

Here is simple [repro](https://github.com/diginikkari/test-runner-errorboundary-issue)

Steps to reproduce:
1. `git clone https://github.com/diginikkari/test-runner-errorboundary-issue`
2. `cd test-runner-errorboundary-issue`
3. `yarn`
4. `yarn storybook`
5. in second terminal: `yarn test-storybook`

### Note!
When I used child component which throwed after mount (with useError) then it wasn't failing for the first story in the file.

**Comment by diginikkari** [comment]
@yannbf I expect that this is could take a while to be solved. So I wonder if there is idea to have configuration to skip certain stories? Something like:
```
Primary.parameters = {
   skipTestRunner: true
}
```
This way more triggier case like this could be skipped until there is some way to tests those and smoke tests could be used in CI.

**Comment by yannbf** [comment]
@diginikkari indeed, I still need to take a look into this! In the meantime you can skip tests by [ejecting the configuration](https://github.com/storybookjs/test-runner#ejecting-configuration) and using `testPathIgnorePatterns`: https://jestjs.io/docs/configuration#testpathignorepatterns-arraystring

**Comment by diginikkari** [comment]
@yannbf thanks for the guide! This worked perfectly 👍. I tried to look for how to skip tests but didn't found it earlier. Now I can run smoke tests and get nice green result 😃

**Comment by yannbf** [comment]
Hey everyone! We just released a test filtering feature in the test-runner v0.15.0, [which you can read here](https://github.com/storybookjs/test-runner#filtering-tests-experimental).

I know this doesn't really fix the issue, but if it's a blocker because these tests fail, you could use the filtering to skip these specific stories for now. We'll get to this at some point! Thanks for your patience <3

**Comment by diginikkari** [comment]
Hi @yannbf thanks for the update. This is already great step forward to be able to skip using tags. You all are doing excellent work with storybook 💪
